playwright.config.ts

```diff
@@ -43,17 +43,31 @@ const config: PlaywrightTestConfig<TestOptions> = {
   /* Configure projects for major browsers */
   projects: [
     {
-      name: "chromium:lxd-5.0-stable",
+      name: "chromium:lxd-5.0-edge",
       use: {
         ...devices["Desktop Chrome"],
-        lxdVersion: "5.0-stable",
+        lxdVersion: "5.0-edge",
```

Comment:
Some tests depend on `5.0-stable`, need to change it there as well to `5.0-edge` get them to adjust behavior on the 5.0 backend: https://github.com/search?q=repo%3Acanonical%2Flxd-ui%205.0-stable&type=code
Or maybe this could be a constant that is used.

Reply:
I've made the `s/5.0-stable/5.0-edge/` changes but I think ideally there would be API extension detection. Unfortunately, that is beyond my coding abilities to implement :/

Reply:
@simondeziel thanks for these changes! Yes we do have API extension detection within the UI components for graceful degradation (see [useSupportedFeatures.tsx](https://github.com/canonical/lxd-ui/blob/main/src/context/useSupportedFeatures.tsx)). Within the test environment, we are using lxd version to execute different tests for specific UI components so that we can ensure the graceful degradation behaves as expected.

Reply:
Ah, good news, thanks!